src/main/java/uk/gov/hmcts/reform/lrdapi/controllers/response/LrdBuildingLocationResponse.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.hmcts.reform.lrdapi.controllers.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode
+public class LrdBuildingLocationResponse {
+
+    @JsonProperty("building_location_id")
+    private String buildingLocationId;
+
+    @JsonProperty("building_location_name")
+    private String buildingLocationName;
+
```

Comment:
if possible try avoid getter & setter in response entities hence you build immutable response

Reply:
Hi @abhijitdiwan `LrdBuildingLocationResponse` is an immutable in nature already as it doesn't have any setters or `@Setter` annotation. We only have `@Getter` annotation on it and having this annotation doesn't alter the nature of this object I believe.

Reply:
Remove @setter on class ..I though it was there.

Reply:
There is no @Setter annotation on the dto class